javascript/builtins/TypedArray.json

```diff
@@ -964,7 +965,7 @@
                 "version_added": null
               },
               "edge": {
-                "version_added": null
+                "version_added": "14"
```

Comment:
IE was set to "10" in #2331, however we both know this is not possible as the feature was added in Edge 14 (I've double-checked this just to be on the safe side).  Perhaps we should update that value here?